docs/deploy.rst

```diff
@@ -0,0 +1,228 @@
+   | Crate.io's Offering to connect Azure's EventHub to CrateDB. | consumer | eventhub-consumer | 1 - 18 (1 - 18 instances) | l      |
+   +-------------------------------------------------------------+----------+-------------------+---------------------------+--------+
+
+The output lists the CrateDB products that are available in your default region.
+The following node sizes (tiers) are available for the deployment of CrateDB
+products: **xs, s, m, l, xl**.
+
+Pick the node size ``xs`` for your test purpose.
+
+It is the smallest node size for a CrateDB product. According to the list above,
+this is the CrateDB cluster ``cratedb.az-gp1``.
+
+
+.. _launch-cluster:
+
+Launch the cluster
+==================
+
+Now that you have created an organization and project, you can deploy the
+cluster by running this command:
+
+.. code-block:: console
+
+   sh$ croud clusters deploy \
+       --product-name cratedb.az-gp1 \
+       --tier xs \
+       --cluster-name my-first-crate-cluster \
+       --project-id 59c7961b-33a2-4478-9e98-2a6b3f54f60e \
+       --version 3.3.0 \
+       --username yourusername \
+       --password "yourpassword" \
+       --region bregenz.a1
+
+.. TIP::
+
+  For an explaining list of the required arguments in this command check out the
+  `Croud CLI Reference`_.
+
+If the operation was successful, you should get the deployment details returned:
+
+.. code-block:: console
+
+   +--------------------------------------+------------------------+----------+-------------+--------------------------------------+----------+------------------------------------------------------+
+   | id                                   | name                   | numNodes | crateVersion| projectID                            | username | fqdn                                                 |
+   +--------------------------------------+------------------------+----------+-------------+--------------------------------------+----------+------------------------------------------------------+
+   | 8d6a7c3c-61d5-11e9-a639-34e12d2331a1 | my-first-crate-cluster |        1 | 3.3.0       | 59c7961b-33a2-4478-9e98-2a6b3f54f60e | default  | my-first-crate-cluster.bregenz.a1.cratedb.net. |
+   +--------------------------------------+------------------------+----------+-------------+--------------------------------------+----------+------------------------------------------------------+
+  ==> Success: Cluster deployed. It may take a few minutes to complete the
+  changes.
+
+.. NOTE::
+
+  The cluster deployment takes around 5-10 minutes to complete.
+
+At the prompt, enter your CrateDB Cloud login credentials.
+
+The administration user interface, or `Admin UI`_, opens and shows your deployed
+CrateDB cluster.
+
+.. image:: _assets/img/admin-ui.png
+
+Congratulations on your first `CrateDB cluster deployment`_.
+
+.. _what-was-learned:
+
+Next steps
+==========
+
+- Getting started with the `Cloud Console`_.
```

Comment:
make these more narrative-y, like suggestions to do a certain thing

check out the wording on https://crate.io/docs/